src/data_management/rasters/RasterLayerTable.tsx

```diff
@@ -165,6 +206,57 @@ export const RasterLayerTable = (props:any) =>  {
           ]}
           newItemOnClick={handleNewRasterClick}
         />
+        { 
+        rowsToBeDeleted.length > 0?
+           <ConfirmModal
+           title={'Are you sure?'}
+           buttonConfirmName={'Delete'}
+           onClickButtonConfirm={() => {
+              deleteFunction && deleteFunction().then(()=>{
+              setRowsToBeDeleted([]);
+              setDeleteFunction(null);
+             });
+           }}
+           cancelAction={()=>{
+            setRowsToBeDeleted([]);
+            setDeleteFunction(null);
+          }}
+          disableButtons={busyDeleting}
+         >
+           
+           <p>Are you sure? You are deleting the following raster layers:</p>
+           
+           {modalDeleteContent(rowsToBeDeleted, busyDeleting)}
+           
+         </ConfirmModal>
+        :
+          null
+        }
+
+        { 
+        rowToBeDeleted?
+           <ConfirmModal
+           title={'Are you sure?'}
+           buttonConfirmName={'Delete'}
+           onClickButtonConfirm={() => {
+             deleteFunction && deleteFunction().then(()=>{
+              setRowToBeDeleted(null);
+              setDeleteFunction(null);
+             });
+             
+           }}
+           cancelAction={()=>{
+             setRowToBeDeleted(null);
+             setDeleteFunction(null);
+           }}
+           disableButtons={busyDeleting}
+         >
+           <p>Are you sure? You are deleting the following raster-layer:</p>
+           {modalDeleteContent([rowToBeDeleted], busyDeleting)}
+         </ConfirmModal>
+        :
+          null
+        }
```

Comment:
I think it is not necessary to treat deletion of 1 raster differently than deletion of multiple rasters. I guess there is technical reason that you implement the deletion separately for 1 raster and multiple rasters?

Reply:
Yes this has technical reasons.
I also thought about this option. I think it is good to investigate, but too much work for now. I made this ticket:
https://github.com/nens/lizard-management-client/issues/556